<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Ustawienia</class>
 <widget class="QDialog" name="Ustawienia">
  <property name="windowModality">
   <enum>Qt::NonModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>445</width>
    <height>378</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>445</width>
    <height>378</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>445</width>
    <height>378</height>
   </size>
  </property>
  <property name="baseSize">
   <size>
    <width>445</width>
    <height>378</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Ustawienia</string>
  </property>
  <property name="sizeGripEnabled">
   <bool>false</bool>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>330</y>
     <width>231</width>
     <height>32</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::SaveAll</set>
   </property>
  </widget>
  <widget class="QGroupBox" name="ustawienia">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>20</y>
     <width>431</width>
     <height>311</height>
    </rect>
   </property>
   <property name="title">
    <string>Ustawienia</string>
   </property>
   <widget class="QCheckBox" name="szuk_w_podk">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>201</width>
      <height>26</height>
     </rect>
    </property>
    <property name="text">
     <string>szukaj w podkaltalogach</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="lista_plikow">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>90</y>
      <width>231</width>
      <height>26</height>
     </rect>
    </property>
    <property name="text">
     <string>zrób listę wszystkich plików</string>
    </property>
   </widget>
   <widget class="QLabel" name="l_sciezk_zapis">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>130</y>
      <width>131</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>ścieżka zapisu listy</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="line_sciezk_zapis">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>160</y>
      <width>281</width>
      <height>33</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="sciezka_zapisz_przycisk">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>160</y>
      <width>98</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Otwórz</string>
    </property>
   </widget>
   <widget class="QLabel" name="l_sciezk_szuk">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>200</y>
      <width>251</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Domyślna ścieżka przeszukiwania</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="line_sciezk_szuk">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>230</y>
      <width>281</width>
      <height>33</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="sciezka_szuk_przycisk">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>230</y>
      <width>98</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Otwórz</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="kryt_szukania">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>20</y>
      <width>171</width>
      <height>111</height>
     </rect>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">QGroupBox#kryt_szukania {
     border-style: solid;
     border-width: 1px;
     border-color: #aaa;
     font: bold 14px;
    
 }</string>
    </property>
    <property name="title">
     <string>Kryteria wyszukiwania</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
    <widget class="QCheckBox" name="rozmiar">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>40</y>
       <width>96</width>
       <height>26</height>
      </rect>
     </property>
     <property name="layoutDirection">
      <enum>Qt::LeftToRight</enum>
     </property>
     <property name="text">
      <string>rozmiar</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="nazwa">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>60</y>
       <width>96</width>
       <height>26</height>
      </rect>
     </property>
     <property name="layoutDirection">
      <enum>Qt::LeftToRight</enum>
     </property>
     <property name="text">
      <string>nazwa</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>340</y>
     <width>181</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Przywróć Domyślne</string>
   </property>
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/icons/settings.png</normaloff>:/icons/settings.png</iconset>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="icons.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>Ustawienia</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>Ustawienia</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
